libc/test/src/stdlib/SortingTest.h

```diff
@@ -28,16 +27,19 @@ class SortingTest : public LIBC_NAMESPACE::testing::Test {
       return -1;
   }
 
-  void test_sorted_array(SortingRoutine sort_func) {
+  static void int_sort(SortFn sort_fn, int *array, size_t array_len) {
+    sort_fn(reinterpret_cast<void *>(array), array_len, sizeof(int),
+            int_compare);
+  }
```

Comment:
if we're going to have different partitioning algorithms based on the size of the input values, we need to make sure to test both of those partitioning algorithms. I'd suggest adding a uint128 test.

Reply:
Good idea, I've added a new test that covers a range of element sizes, `uint128` wouldn't help because it also uses the Lomuto partition.